.github/workflows/ci.yaml

```diff
@@ -926,6 +926,7 @@ jobs:
           - TestMultiNodeAirgapHAInstallation
           - TestSingleNodeAirgapDisasterRecovery
           - TestMultiNodeAirgapHADisasterRecovery
+          - TestSingleNodeNetworkReport
```

Comment:
why not integrate to all existing tests instead of adding a new test for this? different tests can behave differently and would be valuable to have this for all of them

Reply:
Good point, no reason we couldn't do that instead. Happy to make that change!

Reply:
Every test in the suite of tests we have will block merging the PR if it fails though. Would there be future work to integrate it into the other tests and replace this one?

Reply:
Yeah, the intent is that as network reporting gets closer to GA we would expand the usage of it in testing